rfc-005-template-contributing-process.md

```diff
@@ -0,0 +1,154 @@
+Additionally, a large, inclusive group of volunteer community reviewers will review templates that are in the workshop phase.
+A small group of experienced pull request reviewers will review and check the templates for quality before merging them into the `templates` repository.
+
+Templates will be written in six different phases with certain checkpoints and milestones that each template must meet along the way:
+
+1. **Join the community** - A new contributor joins the community and find an initiative they want to work on.
+
+2. **Adopt a template** - If they are interested in contributing templates, they join a template writing working group led by a template mentor. They decide which template set to begin working on.
+
+3. **Research and draft** - They research best practices and look at examples of the type of document they are working on. When they are finished, they will create a rough draft.
+
+4. **Workshop the drafts with the community** - Revise and refine their template set with the community pull reviewers. At least three community reviews are required to move to the next phase.
+
+5. **Submit a pull request** - They open a pull request and it is reviewed for quality by the pull request reviewers. At least two pull request reviewer approvals are needed to be merged into the repository.
+
+6. **Improve the template with user feedback** - As users try the template in the wild, they may report usability issues or provide feedback for improvements to the template. Either the original template author or another templateer will evaluate feedback and incorporate it into future versions of the template. When that happens, the template will go through the previous phases again.
+
+Each template set will include:
+- **Raw template** - The basic template for the type of document you are creating. It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.
+- **Template guide** - Provides a deeper explanation of how to fill in the raw template. It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document. Additional content may be included as needed.
+- **Before you start (optional)** - This optional guide is designed to help template users understand some of the key research, brainstorming, or pre-writing steps that they might need to do before adding this type of document to a guide. If this documentation type has certain key decisions that need to be made with stakeholders or if it has a dependency on some other type of document or process, that should be mentioned in this guide.
+- **Template example (optional)** - The Good Docs Project is in the process of designing a fake documentation project that can provide examples of our templates in action. Until this project is ready, template authors may optionally create an example of the template in action.
+
+
+## Links and prior art

```

Comment:
Perhaps this is just a term I'm unfamiliar with, but I'm not sure what "prior art" means in this context. "Prior work"?

Reply:
This term was added to the RFC template per RFC-000 at Morgan's suggestion. I believe the term "prior art" comes from US patent law and generally refers to any kind of designs or other intellectual property that contributed to the final proposal.

Reply:
I'm not going to take action on this comment because it's not relevant to the scope of the RFC-005 and is about the RFC process itself. My recommendation would be that if we have objections to the term "prior art" that one of you opens a pull request to change the RFC template.